src/Microsoft.AspNetCore.Sockets.Client/LongPollingTransport.cs

```diff
@@ -28,10 +29,14 @@ public class LongPollingTransport : ITransport
 
         public Task Running { get; private set; }
 
+        public LongPollingTransport(HttpClient httpClient)
```

Comment:
Its nice for testing though so we don't have to create a logger factory each time we want to use Long Polling. I'll remove it though